### PR TITLE
make use of a 'default' response [closes #96]

### LINKED ISSUE #96: support `responses.default`
According [to the responses spec](http://swagger.io/specification/#responsesObject), a `default` key should be used to validate any response codes not explicitly declared. Right now, though, flex will raise `invalid_status_code` without checking for the presence of `default`.

This should be an easy fix: the check for default can be done [here](https://github.com/pipermerriam/flex/blob/a8ddfae3393aea596b390022bb7263d2ca408937/flex/validation/response.py#L46).

### PR DESCRIPTION
**Pull request by simon-weber**
See #96.¹

¹ see also [this cute picture of a hedgehog](https://s-media-cache-ak0.pinimg.com/564x/47/31/0e/47310ecbfc266abbea3db0cd626fd0dc.jpg).

### CODE REVIEW

File: tests/validation/response/test_response_status_code_validation.py [hunk 1]
```diff
@@ -43,3 +43,31 @@ def test_response_parameter_validation():
...
+                        'default': {'description': 'Unexpected error.'}
+                    },
+                },
+            },
+        },
+    )
+
+    response = ResponseFactory(url='http://www.example.com/get', status_code=301)
+
+    validate_response(
```
Comment (simon-weber) [inline tests/validation/response/test_response_status_code_validation.py]: do you have a preference for a Failure rather than an Error here?

Comment (pipermerriam) [inline tests/validation/response/test_response_status_code_validation.py]: If you are referring to assertion vs exception, I do not.  As long as the test suite will not pass if this test is failing.

**Comment by pipermerriam** [comment]
:+1: looks good.

**Comment by pipermerriam** [comment]
I'm going to give it a bit to see if #97 gets PR'd and fixed today.  I'll get you a release with these changes by the EOD either way.

**Comment by simon-weber** [comment]
No need to rush if it's inconvenient; we can just point to our fork for now.

**Pull request merged**